web-server/v0.4/src/app.js

```diff
@@ -7,13 +7,13 @@ import storage from 'redux-persist/lib/storage';
  * `dashboard` and `search` are blacklisted from the saved state as the namespaces
  * persist data that is constantly updated on the server side.
  */
-const appPath = window.location.pathname.split('/').pop();
```

Comment:
This means we can't load different versions of the dashboard, right?

I'd like to be able to handle data migrations by loading two different configurations.

Reply:
This does allow the deployment of multiple versions of the dashboard; however, the `appName` field needs to be changed for each deployment so each deployment reads from their respective local storage key. I've added details for accomplishing this with the changes to the `README.md`.

Reply:
Not sure, but that looks like a code change before you build the application.  Why not have the `appName` read from the on-disk configuration file with everything else?

Reply:
I agree! Should be introduce a new configuration file for app specific configuration fields? We're only referencing datastore specific fields in the current `config.json.j2` file.

Reply:
We can introduce a new configuration file, but that would just be like using the window location as well, since each app would have its own path that way.

Why do we have to remove the `window.location.pathname.split('/').pop()` method?

Reply:
Referencing the last slash separated path item in the url causes multiple local Redux stores to be created since there are many routes within the app (especially in the case of URL session generation where we create a unique ID per session). 

We can simply reference the first path item to avoid this behavior - this will also maintain the naming logic when we deploy the app.